native/.gitignore

```diff
@@ -35,7 +35,7 @@ yarn-error.log*
 .hprof
 
 # CocoaPods
-/ios/Pods/
+/**/Pods/
```

Comment:
❓ In case the pods are installed elsewhere?

Reply:
Hmm I guess, this change was suggested by the update helper. Definitely should not hurt though.